dnf/util.py

```diff
@@ -63,7 +63,11 @@ def _parse_specs(namespace, values):
     setattr(namespace, "filenames", [])
     setattr(namespace, "grp_specs", [])
     setattr(namespace, "pkg_specs", [])
+    tmp_set = set()
     for value in values:
+        if value in tmp_set:
```

Comment:
this is not needed at all. In set, all elements are unique. Everything what you need is: `for value in set(values)`

Reply:
> this is not needed at all. In set, all elements are unique. Everything what you need is: `for value in set(values)`

True, but I want to keep order of arguments. Specially for logging reason.